### PR TITLE
Ftc wizard

### PR DESCRIPTION
**Pull request by lukefrogger**
## What's here
This is an update to the FTC wizard to complete the following:
1. Update the wizard with new styles
2. Complete the wizard build-out
3. Add required WME components for the wizard

### Refactoring
This includes a few refactors to help future developers be happy 😄 
- `/contexts/FirstTimeConfigurationProvider.tsx`: Centralized the function types so there is one place to modify arguments for functions instead two.
- `wizards/wizardWrapper`: The header and content for wizard are now handled within each wizard index file. This makes controlling the component visibility and page layout much easier. 

### New & updated components
__Autocomplete__`(new)`:  A text input with select options that are searchable by typing in the input.
__Chips-Input__`(new)`: A text input that allows users to type in text and saves it in a chip within the input.
__Template-Select-Item__`(updated)`: Added the ability to pass a URL to the component that will display an iframe with the site embedded.

### Screenshots
gif-ftc

#### chips-input
Screenshot 2023-03-20 at 9 06 48 AM

#### autocomplete
Screenshot 2023-03-20 at 9 06 59 AM

### CODE REVIEW

File: packages/sitebuilder/src/contexts/FirstTimeConfigurationProvider.tsx [hunk 1]
```diff
@@ -150,7 +195,8 @@ const FirstTimeConfigurationProvider = ({
 				resetFormValue,
 				setIsLoading,
 				setLogoValue,
-				validateUsernamePassword
+				validateUsernamePassword,
+				shouldBlockNextStep
```
Comment (chumpgrub) [inline packages/sitebuilder/src/contexts/FirstTimeConfigurationProvider.tsx:199]: I like naming of this prop

**Review by chumpgrub — COMMENTED** [review]
Wow! Lot's of great work, Luke! Could you address the error for the Base story of Chipinput before you merge this, please?

**Comment by aykanburcak** [comment]
Awesome work @lukefrogger 👏🏼

File: packages/sitebuilder/src/wizards/first-time-configuration/screens/StyleSelection.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import { useFirstTimeConfiguration } from '@sb/hooks';
+import { TemplateSelectGroup, TemplateSelectItem } from '@moderntribe/wme-ui';
+import { Box } from '@mui/material';
+
+const templateData = [
+	{
+		image: 'http://localhost:8888/wp-content/uploads/2023/03/template-preview.png',
```
[the remark this answers is not in the document]
Comment (lukefrogger) [reply packages/sitebuilder/src/wizards/first-time-configuration/screens/StyleSelection.tsx:7]: The image is really big which ballons the bundle size and using a local image also requires webpack changes that won't be required when we go live. For those reasons I'm going to leave it as a static import for now.

**Pull request merged**